scvid/data/dadc_dataset.py

```diff
@@ -104,6 +104,7 @@ def __getitem__(self, idx: int | list[int] | slice) -> dict[str, np.ndarray]:
 
         data = {}
         data["X"] = X.toarray() if issparse(X) else X
+        data["obs_names"] = self.dadc[idx].obs_names.values.astype(int)
```

Comment:
Could could comment on this?

Reply:
These are CAS IDs that are used during writing embeddings to files.

Reply:
Why convert to int? What if CAS IDs are non-int?

Reply:
I think it is a convention for CAS, right @fedorgrab ?

Returned data dictionary is collated and before that it is converted to a `torch.Tensor`. So it has to be a number. And I guess the only number type that makes sense for an ID is an `int` type.